packages/skia/src/skia/types/Paragraph/TextStyle.ts

```diff
@@ -53,7 +53,7 @@ export interface SkTextStyle {
   decoration?: number;
```

Comment:
should decoration be of type `TextDecoration` here?

Reply:
yeah exactly, I overlooked it. thanks!